### PR TITLE
feat(esl-panel-group): support for esl-panel-group driven animation to closed state

### PR DESCRIPTION
**Pull request by ala-n**
Closes: #1789

### CODE REVIEW

File: src/modules/esl-panel-group/README.md [hunk 1]
```diff
@@ -20,18 +20,19 @@ ESLPanelGroup.register();
 - `mode-cls` - rendering mode class pattern (default: `esl-{mode}-view`). Uses ESLUtils `format` syntax for `mode` placeholder
 - `mode-cls-target` - Element [ESLTraversingQuery](../esl-traversing-query/README.md)  selector to add class that identifies mode (ESLPanelGroup itself by default)
 - `animation-class` - class(es) to be added during animation ('animate' by default)
-- `no-animate` - list of breakpoints to disable collapse/expand animation (for both Group and Panel animations)
+- `no-animate` - list of breakpoints to skip collapse/expand animation (for both Group and Panel animations)
 - `refresh-strategy` - defines behaviour of active panel(s) in case of configuration change:
   * `initial` - activates initially opened panel(s)
-  * `last` - maintain currently active panel(s) open
+  * `last` - maintain a currently active panel(s) open
```
Comment (julia-murashko) [inline src/modules/esl-panel-group/README.md:26]:
```suggestion
  * `last` - maintains a currently active panel(s) open
```

**Review by julia-murashko — APPROVED** [review]

File: src/modules/esl-panel-group/core/esl-panel-group.ts [hunk 2]
```diff
@@ -294,18 +295,23 @@ export class ESLPanelGroup extends ESLBaseElement {
...
   /** Process {@link ESLPanel} show event */
-  @listen('esl:show')
-  protected _onShow(e: CustomEvent): void {
+  @listen('esl:show esl:hide')
+  protected _onStateChanged(e: CustomEvent): void {
     const panel = e.target;
     if (!this.includesPanel(panel)) return;
+    this.toggleAttribute('has-opened', panel.open);
+
     if (this.currentMode !== 'tabs') return;
```
Comment (NastaLeo) [inline src/modules/esl-panel-group/core/esl-panel-group.ts:309]: Sorry, I missed something, panel in accordion won't be animated anymore?

Comment (ala-n) [reply src/modules/esl-panel-group/core/esl-panel-group.ts:309]: Nope, in accordion, it is animated as usual, but the animation is handled by ESLPanels itself.

**Review by dshovchko — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

**Review by HenadzV — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

**Pull request merged**